### PR TITLE
[IMP] e-commerce : added custom fields to product creation modal

### PR DESCRIPTION
**Pull request by luve-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/loyalty/views/loyalty_program_views.xml [hunk 1]
```diff
@@ -48,14 +48,14 @@
...
-                                        When generating coupon, you can define a specific points value that can be exchanged for rewards. 
+                                        When generating coupon, you can define a specific points value that can be exchanged for rewards.
                                     </div>
                                 </p>
                                 <p class="text-muted" attrs="{'invisible': [('program_type', '!=', 'loyalty')]}" colspan="2">
                                     When customers make an order, they accumulate points they can exchange for rewards on the current order or on a future one.
                                 </p>
                                 <p class="text-muted" attrs="{'invisible': [('program_type', '!=', 'promotion')]}" colspan="2">
-                                    Set up conditional rules on the order that will give access to rewards for customers                                            
+                                    Set up conditional rules on the order that will give access to rewards for customers
```
Comment (Williambraecky) [inline addons/loyalty/views/loyalty_program_views.xml:58]: We can do these changes in another PR if you wish but this is not the place to do so.

File: addons/website_sale/views/product_product_add.xml [hunk 2]
```diff
@@ -6,9 +6,19 @@
...
         <form js_class="website_new_content_form">
-            <group>
+            <group name="pricing">
                 <field name="website_url" invisible="1"/>
+                <field name="company_id" invisible="1"/>
+                <field name="currency_id" invisible='1'/>
+                <field name="cost_currency_id" invisible="1"/>
                 <field name="name" placeholder="e.g. Cheese Burger" string="Product Name"/>
+                <label for="list_price" class="mt-1"/>
+                <div name="pricing">
```
Comment (Williambraecky) [inline addons/website_sale/views/product_product_add.xml:16]: Is the name actually relevant? Shouldn't it use class="o_row"?

**Review by Williambraecky — CHANGES_REQUESTED** [review]

File: addons/website_sale/views/product_product_add.xml (hunk 2 above)
Comment (luve-odoo) [reply addons/website_sale/views/product_product_add.xml]: Well I changed to name to make it more explicit, but for the class, according to the spec, the 2 fields should one above the other, so the <mt-1> class is I think mandatory. But indeed, the <span class="ms-2"> was not necessary. Tell me if it is good for u !

**Review by Williambraecky — APPROVED** [review]
Thanks
robodoo r+
